evaluate/repository.go

```diff
@@ -139,10 +147,19 @@ func contextForTask(taskIdentifier task.Identifier, language language.Language,
 	}
 
 	switch taskIdentifier {
+	case task.IdentifierCodeRepair:
+		mistakes, err := language.Mistakes(logger, repositoryPath)
+		if err != nil {
+			return context, err
+		}
+
+		context.Arguments = mistakes
+
+		return context, nil
```

Comment:
If there are no mistakes we don't need to execute the task at all. Should we also return a boolean indicating if a task should be skipped?